capa/ida/plugin/model.py

```diff
@@ -602,8 +604,8 @@ def render_capa_doc_feature(
 
         # wb: 614: substring feature?
         elif isinstance(feature, (frzf.RegexFeature, frzf.SubstringFeature)):
-            for capture, locations in sorted(match.captures.items()):
-                if location in locations:
+            for capture, addrs in sorted(match.captures.items()):
+                if location in [addr.value for addr in addrs]:
```

Comment:
do you recall what type `addr` is? i think we should avoid reaching into its properties unless we assert its type (for documentation and bug finding)

Reply:
090d379